TEMPLATES.md

```diff
@@ -109,5 +109,5 @@ Comma-separated list of container names for which ci-operator will wait until co
 to gather the artifacts.
 
 `ci-operator.openshift.io/containers-logged-on-failure`:
-Comma-separated list of container names for which ci-operator will collect and log their output if pod fails.
-By default, only the logs from the failed containers will be output.
\ No newline at end of file
+Comma-separated list of container names for which ci-operator will collect and log their output to artifacts if pod fails.
+The artifact name will be generated in the format of `$container_name-out.log`.
```

Comment:
What is the chance of a name clash with a file that the test will want to put there? This will lead to filenames like `setup-out.log`, not that unique. Should we prefix this, or maybe put it into a separate directory?

Reply:
Yes, this is a possibility that I also consider, that's why I wanted to specify the name in the docs. So the user will avoid writing his script to output to a file with the same name. It can be avoided also programmatically, but this will lead in an extra code as well.

Reply:
If we reserve a filename this way, it should be something more specific. IMHO.

Reply:
Will it be better, if we send them in a different directory, inside pod's directory of course, with the $containername.log format?

Reply:
Like having something like `container-logs/`? SGTM